tekton/cronjobs/bases/nightly-tests/trigger-nightly-test.yaml

```diff
@@ -64,4 +70,12 @@ spec:
                 value: "s390x"
               - name: KUBECONFIG_SECRET
                 value: "secret"
+              - name: REMOTE_SECRET
```

Comment:
NIT: maybe the variables could be named `KUBECONFIG_SECRET_NAME` `REMOTE_SECRET_NAME`

Reply:
updated